Mathlib/MeasureTheory/Function/UnifTight.lean

```diff
@@ -214,7 +214,8 @@ and `tendstoInMeasure_of_tendsto_eLpNorm`. -/
 
 /-- Intermediate lemma for `unifTight_of_tendsto_Lp`. -/
 private theorem unifTight_of_tendsto_Lp_zero (hp' : p ≠ ∞) (hf : ∀ n, MemLp (f n) p μ)
-    (hf_tendsto : Tendsto (fun n ↦ eLpNorm (f n) p μ) atTop (𝓝 0)) : UnifTight f p μ := fun ε hε ↦by
+    (hf_tendsto : Tendsto (fun n ↦ eLpNorm (f n) p μ) atTop (𝓝 0)) :
+    UnifTight f p μ := fun ε hε ↦ by
```

Comment:
How about making these lines
```suggestion
    (hf_tendsto : Tendsto (fun n ↦ eLpNorm (f n) p μ) atTop (𝓝 0)) : UnifTight f p μ := by
  intro ε hε
```
instead?

Reply:
that suggestion lgtm